src/util/teamsUtil.js

```diff
@@ -24,10 +103,13 @@ export async function fetchTeams(apollo, sortOption, category, membershipType, q
 	}
 }
 
-export async function fetchLeaderboard(apollo) {
+export async function fetchLeaderboard(apollo, category) {
```

Comment:
Looks like this file needs unit tests in general. Doesn't need to happen now, but it would be a good addition.